tests/Integration/Integrations/FacebookInstantArticlesTest.php

```diff
@@ -17,7 +17,9 @@
  * Test Facebook Instant Articles integration.
  */
 final class FacebookInstantArticlesTest extends TestCase {
-	public const REGISTRY_KEY = 'parsely-analytics-for-wordpress';
+	// This values should mirror the values in `class-facebook-instant-articles.php`.
+	private const REGISTRY_IDENTIFIER   = 'parsely-analytics-for-wordpress';
+	private const REGISTRY_DISPLAY_NAME = 'Parse.ly Analytics';
```

Comment:
> This values should mirror the values in `class-facebook-instant-articles.php`

This is a code smell.

If no other application logic needs to access the constant, then let's make them private in `class-facebook-instant-articles.php`, scrap these test class constants, and use Reflection to grab the constants from the class under test instead.

- https://www.php.net/manual/en/reflectionclass.getreflectionconstants.php
- https://www.php.net/manual/en/reflectionclass.getconstants.php